content/docs/using-tina-editor.md

```diff
@@ -43,6 +46,6 @@ Once saved, the changes to the document will be stored against the files.
 
 There's plenty to do to customize your editing experience. We suggest:
 
-- Checking out [our concept docs](/docs/schema/), to learn how Tina powers the starters under the hood.
-- Learn how [Tina can be extended](/docs/advanced/extending-tina/) to create new field components
-- Make your site [editable with Tina on production](/docs/tina-cloud/)
+* Checking out [our concept docs](/docs/schema/), to learn how Tina powers the starters under the hood.
```

Comment:
**nitpick**: Remove full-stop for consistency with other dot points.
```suggestion
* Checking out [our concept docs](/docs/schema/), to learn how Tina powers the starters under the hood
```

Reply:
✅🙏